README.md

```diff
@@ -1 +1,17 @@
-# python-swmrtools
+# swmr_tools
+
+swmr_tools is a Python library developed by Diamond Light Source ltd designed to facilitate reading and processing data from nexus (hdf5) files as they are being written
+
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install swmr_tools
+
+'''bash
```

Comment:
Wrong quotes - should be back ticks (\')

Reply:
Changed to back ticks ... I was wondering why it hadn't displayed properly.